integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/WebhookSiteSteps.java

```diff
@@ -32,4 +35,17 @@ public void webhookSiteContainsRequest(String requestText, int timeoutMinutes) {
                                 requestTextWithoutPlaceholders)
                         .anyMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders)));
     }
+
+    @Then("^Webhook site does not contains request with text \"([^\"]*)\" within (\\d+) seconds$")
+    public void webhookSiteDoesNotContainsRequest(String requestText, long timeoutSeconds) throws InterruptedException {
+        String requestTextWithoutPlaceholders = ContextResolver.resolveWithScenarioContext(context, requestText);
+        BooleanSupplier failureCondition =
+                () -> WebhookSiteResource.requests().stream().map(WebhookSiteRequest::getContent).noneMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders));
+        Duration duration = Duration.ofSeconds(timeoutSeconds);
+        Instant timeoutTime = Instant.now().plus(duration);
+        while (timeoutTime.isAfter(Instant.now())) {
+            Thread.sleep(duration.toMillis());
+            assertThat(failureCondition.getAsBoolean()).as("Searching for request containing text: '%s'", requestTextWithoutPlaceholders).isTrue();
```

Comment:
Can be rewritten to something like:
```
            assertThat(WebhookSiteResource.requests())
                    .map(WebhookSiteRequest::getContent)
                    .as("Checking that WebHook site doesn't contain request containing text: '%s'", requestTextWithoutPlaceholders)
                    .noneMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders));
```
That way the assertion has access to list of requests so in case of failure it can provide all the requests it searched as a context for test analysis.